Air_Traffic_Control_System/src/Aircraft.h

```diff
@@ -7,19 +7,33 @@
 
 #ifndef AIRCRAFT_H_
 #define AIRCRAFT_H_
+#include <pthread.h>
+#include <vector>
+
+using namespace std;
+
 
 class Aircraft {
 public:
 	Aircraft();
+	Aircraft(int time_at_boundary, int flight_level, int flight_id, int posX, int posY, int posZ,int speedX,int speedY,int speedZ);
+	void UpdateAircraftPosition();
+	void updatePositionX();
+	void updatePositionY();
+	void updatePositionZ();
+	void updateFlightLevel();
+	void ServiceInterrogationSignal();
+	void receiveInterrogationSignal();		//this blocks until interrogation signal arrives
+	char collectTransponderData();
+	void senTransponderData(char transponderData[]);
+
 	virtual ~Aircraft();
 
 private:
-	int angle;
-	int distance;
-	int flightId;
-	int flightLevel;
-	int speed;
-	int position;
+	int time_at_boundary, flight_level, flight_id;
```

Comment:
Seperate these into 3 different lines since they aren't really apart of the same group like speed or position.